### DISCUSSION TITLE
Al crear una factura, ¿a qué se refiere el concepto de un documento y por qué es obligatorio?

### DISCUSSION THREAD

**Discussion by JuliusLancelot** [AndresRamos/ARSoftware.Contpaqi.Comercial.Api#60] [Q&A] [answered]
Hay un "concepto" en el modelo Documento. Al intentar crear un documento (una factura) da el error de que no existe el concepto del documento. Pero no entiendo muy bien a qué se refiere ese concepto. 

"documento": {
      "id": 0,
      **"concepto": {
        "id": 0,
        "codigo": "FACTURAPRUEBA",
        "nombre": "",
        "datosExtra": {}**
      }
}
El resultado es este:  "errorMessage": "Mensaje:[Concepto documento no existe.] Numero:[120146]",

**Comment by JuliusLancelot** [comment, marked as answer]
Navegando en las opciones de Contpaq encontré que el concepto se refiere a si el documento es Factura, Factura a Crédito, Cotización, Devolución, etc.

**Comment by AndresRamos** [comment]
Los conceptos de documentos puedes interpretarlos como los tipo de documento. Tenemos documentos de tipo Compra, Factura, Pagos y asi. Cuando creas una factura, o compra, estas generando un documento. En Comercial todos son documentos, pero para diferenciar necesitamos asignarle el concepto. 

En la solicitud, solo es necesario asignar el `Codigo` del `Concepto`.
